monkey/monkey_island/cc/services/config_schema/monkey.py

```diff
@@ -67,7 +67,8 @@
                         "HiddenFiles",
                         "TrapCommand",
                         "ChangeSetuidSetgid",
-                        "ScheduleJobs"
+                        "ScheduleJobs",
+                        "ClearCommandHistory"
```

Comment:
Of all the techniques, this one seems the most potentially "harmful". Let's not make it `off` by default?